specs/darwin/crash_logs.table

```diff
@@ -0,0 +1,27 @@
+table_name("crash_logs")
+description("Application Crash Logs")
+schema([
+    Column("process", TEXT, "Name of the process that crashed."),
+    Column("application_path", TEXT, "Path to the crashed process."),
+    Column("log_location", TEXT, "Location of log file."),
+    Column("identifier", TEXT, "Identifier of the crashed process."),
+    Column("version", TEXT, "Version info of the crashed process."),
+    Column("parent_proc", TEXT, "Parent of the crashed process."),
+    Column("responsible", TEXT, "Process responsible for the crashed process."),
+    Column("user_id", TEXT, "User ID of the crashed process."),
+    Column("date_time", TEXT, "Date/Time at which the crash occurred."),
+    Column("crashed_thread", TEXT, "Thread ID which crashed."),
+    Column("stack_trace", TEXT, "Most recent frame from the stack trace."),
+    Column("exception_type", TEXT, "Exception type of the crash."),
+    Column("exception_codes", TEXT, "Exception codes from the crash."),
+    Column("exception_notes", TEXT, "Exception notes from the crash."),
+    Column("rax", TEXT, "The value of the rax register."),
+    Column("rbx", TEXT, "The value of the rbx register."),
+    Column("rcx", TEXT, "The value of the rcx register."),
+    Column("rdx", TEXT, "The value of the rdx register."),
+    Column("rdi", TEXT, "The value of the rdi register."),
+    Column("rsi", TEXT, "The value of the rsi register."),
+    Column("rbp", TEXT, "The value of the rbp register."),
+    Column("rsp", TEXT, "The value of the rsp register."),
```

Comment:
That might be a good idea, but my thoughts with doing a TEXT value was that I usually find more value in seeing a register represented as a hex string, as opposed to an int. I'm open to thoughts with this though, it should be just casting the value to an int as I store it.